src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -152,5 +153,13 @@ public string CookieName
         /// to the client. This can be used to mitigate potential problems with very large identities.
         /// </summary>
         public IAuthenticationSessionStore SessionStore { get; set; }
+
+        CookieAuthenticationOptions IOptions<CookieAuthenticationOptions>.Value
```

Comment:
But why not providing a wrapper implementing `IOptions<T>`? 

    public class Options<T> : IOptions<T> where T : class, new() {
        public Options(T value) {
            Value = value;
        }

        public T Value { get; }
    }